app/src/main/java/com/example/mycleaninglog/ExpandableCard.kt

```diff
@@ -21,16 +21,13 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.alpha
 import androidx.compose.ui.draw.rotate
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextOverflow
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.TextUnit
 import androidx.compose.ui.unit.dp
-import com.example.mycleaninglog.dto.cleaningTask
-import com.example.mycleaninglog.dto.myRoom
+import com.example.mycleaninglog.dto.CleaningTask
+import com.example.mycleaninglog.dto.MyRoom
```

Comment:
Good suggestion, though it was already covered in a previous review.  When that occurs, first in wins.